Actionling (shellcheck) failures -- interesting they don't appear introduced by the zizmor changes, so maybe spellcheck wasn't run before?
```
.github/workflows/benchmarks_report.yml:61:9: shellcheck reported issue in this script: SC2016:info:1:8: Expressions don't expand in single quotes, use double quotes for that [shellcheck]
   |
61 |         run: |
   |         ^~~~
.github/workflows/docker-publish.yml:101:9: shellcheck reported issue in this script: SC2086:info:1:40: Double quote to prevent globbing and word splitting [shellcheck]
    |
101 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:145:9: shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting [shellcheck]
    |
145 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:145:9: shellcheck reported issue in this script: SC2086:info:2:36: Double quote to prevent globbing and word splitting [shellcheck]
    |
145 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:145:9: shellcheck reported issue in this script: SC2086:info:3:36: Double quote to prevent globbing and word splitting [shellcheck]
    |
145 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:145:9: shellcheck reported issue in this script: SC2086:info:4:36: Double quote to prevent globbing and word splitting [shellcheck]
    |
145 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:156:9: shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting [shellcheck]
    |
156 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:156:9: shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting [shellcheck]
    |
156 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:169:9: shellcheck reported issue in this script: SC2086:info:1:32: Double quote to prevent globbing and word splitting [shellcheck]
    |
169 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:180:9: shellcheck reported issue in this script: SC2086:info:1:32: Double quote to prevent globbing and word splitting [shellcheck]
    |
180 |         run: |
    |         ^~~~
.github/workflows/reusable_run_tox_test.yml:191:9: shellcheck reported issue in this script: SC2086:info:1:32: Double quote to prevent globbing and word splitting [shellcheck]
    |
191 |         run: |
    |         ^~~~
```

--- a/.github/workflows/benchmarks_report.yml
+++ b/.github/workflows/benchmarks_report.yml
@@ -59,7 +59,7 @@ jobs:
 
       - name: Replace URLs
         run: |
-          sed -i 's@||BENCHMARK_CI_LOGS_URL||@${GITHUB_EVENT_WORKFLOW_RUN_HTML_URL}@g' message.txt
+          sed -i "s@||BENCHMARK_CI_LOGS_URL||@${GITHUB_EVENT_WORKFLOW_RUN_HTML_URL}@g" message.txt
         env:
           GITHUB_EVENT_WORKFLOW_RUN_HTML_URL: ${{ github.event.workflow_run.html_url }}
 

--- a/.github/workflows/docker-publish.yml
+++ b/.github/workflows/docker-publish.yml
@@ -99,6 +99,6 @@ jobs:
 
       - name: Test Docker image
         run: |
-          docker run --rm --entrypoint=/bin/bash ${STEPS_DOCKER_BUILD_OUTPUTS_IMAGEID} -ec "python3 -m napari --version"
+          docker run --rm --entrypoint=/bin/bash "${STEPS_DOCKER_BUILD_OUTPUTS_IMAGEID}" -ec "python3 -m napari --version"
         env:
           STEPS_DOCKER_BUILD_OUTPUTS_IMAGEID: ${{ steps.docker_build.outputs.imageid }}

--- a/.github/workflows/reusable_run_tox_test.yml
+++ b/.github/workflows/reusable_run_tox_test.yml
@@ -143,10 +143,10 @@ jobs:
         # FOURTH=none
         shell: bash
         run: |
-          python tools/split_qt_backend.py 0 ${INPUTS_QT_BACKEND} >> "$GITHUB_ENV"
-          python tools/split_qt_backend.py 1 ${INPUTS_QT_BACKEND} >> "$GITHUB_ENV"
-          python tools/split_qt_backend.py 2 ${INPUTS_QT_BACKEND} >> "$GITHUB_ENV"
-          python tools/split_qt_backend.py 3 ${INPUTS_QT_BACKEND} >> "$GITHUB_ENV"
+          python tools/split_qt_backend.py 0 "${INPUTS_QT_BACKEND}" >> "$GITHUB_ENV"
+          python tools/split_qt_backend.py 1 "${INPUTS_QT_BACKEND}" >> "$GITHUB_ENV"
+          python tools/split_qt_backend.py 2 "${INPUTS_QT_BACKEND}" >> "$GITHUB_ENV"
+          python tools/split_qt_backend.py 3 "${INPUTS_QT_BACKEND}" >> "$GITHUB_ENV"
         env:
           INPUTS_QT_BACKEND: ${{ inputs.qt_backend }}
 
@@ -154,9 +154,9 @@ jobs:
         timeout-minutes: ${{ inputs.timeout }}
         shell: bash
         run: |
-          echo ${MAIN}
+          echo "${MAIN}"
           tox --version
-          python -m tox run --installpkg ${WHEEL_PATH} -- --basetemp=.pytest_tmp
+          python -m tox run --installpkg "${WHEEL_PATH}" -- --basetemp=.pytest_tmp
           rm -r .tox
         env:
           BACKEND: ${{ env.MAIN }}
@@ -167,7 +167,7 @@ jobs:
         if : ${{ env.SECOND != 'none' }}
         shell: bash
         run: |
-          python -m tox run --installpkg ${WHEEL_PATH} -- --basetemp=.pytest_tmp
+          python -m tox run --installpkg "${WHEEL_PATH}" -- --basetemp=.pytest_tmp
           rm -r .tox
         env:
           BACKEND: ${{ env.SECOND }}
@@ -178,7 +178,7 @@ jobs:
         if : ${{ env.THIRD != 'none' }}
         shell: bash
         run: |
-          python -m tox run --installpkg ${WHEEL_PATH} -- --basetemp=.pytest_tmp
+          python -m tox run --installpkg "${WHEEL_PATH}" -- --basetemp=.pytest_tmp
           rm -r .tox
         env:
           BACKEND: ${{ env.THIRD }}
@@ -189,7 +189,7 @@ jobs:
         if: ${{ env.FOURTH != 'none' }}
         shell: bash
         run: |
-          python -m tox run --installpkg ${WHEEL_PATH} -- --basetemp=.pytest_tmp
+          python -m tox run --installpkg "${WHEEL_PATH}" -- --basetemp=.pytest_tmp
           rm -r .tox
         env:
           BACKEND: ${{ env.FOURTH }}

